packages/react/src/textfield/TextField.tsx

```diff
@@ -99,7 +99,7 @@ function TextField(props: TextFieldProps, ref: Ref<HTMLInputElement>) {
   );
 }
 
-function Divider({ className }: { className: string }) {
+export function Divider({ className }: { className: string }) {
   return (
     <span className={cx(className, 'block h-6 w-px flex-none bg-black')} />
   );
```

Comment:
Burde denne kanskje blitt dratt ut herfra? Enten ut til `src/textfield/Divider.tsx` eller til en helt egen dings. Føles litt rart å skulle importere den fra `TextField.tsx` (i praksis).

Reply:
Ja, var inne litt på tanken selv @ingara. Men prøver å være litt forsiktig med å endre for mye på eksisterende ting nå som jeg er helt ny 😅 Akkurat denne her er jo ganske tett knyttet opp mot akkurat input-felter så jeg tenker kanskje at `src/textfield/Divider.tsx` kan funke fint? Blir kanskje også litt rart å la den stå helt for seg selv når den kun brukes i input-felter? Men det er klart at man kan tenke seg att det kommer andre komponenter som kommer til å ha en vertikal Divider i seg på sikt. Kan prøve å se litt i Figma i morgen.

Man kunne jo for så vidt også tenkt seg at man grupperer `TextField` og `NumberField` i én mappe (f.eks. `fields`) og putter `Divider` sammen med de der. Men får man en mappestruktur som kanskje bryter litt med resten, og det ønsker vi kanskje heller ikke.